src/app/news/news-list.component.ts

```diff
@@ -20,6 +20,7 @@ import { DialogsPromptComponent } from '../shared/dialogs/dialogs-prompt.compone
 export class NewsListComponent {
 
   @Input() items: any[] = [];
+  @Input() editSuccessMessage: 'News has been updated successfully.';
```

Comment:
This isn't quite correct.  What you've done here is initialize a property that is undefined that is type limited to only accept the value `'News has been updated successfully.` in an assignment.

The line should be
```
@Input() editSuccessMessage = 'News has been updated successfully.';
```

Sorry I missed this last night, but when assigning a value to a variable we have a lint rule that rejects the code if there is a type assignment that matches the implied type based on the value of the variable.